src/main/java/com/google/firebase/auth/UserRecord.java

```diff
@@ -324,6 +357,19 @@ public UpdateRequest setEmail(String email) {
     }
 
     /**
+     * Updates the phone number associated with this user.
+     *
+     * @param phone a valid phone number string or null.
+     */
+    public UpdateRequest setPhoneNumber(String phone) {
```

Comment:
Do we not want to use Nullable here? (and everywhere else).

I feel we shouldn't be calling this out in our comments but rather in our code. Furthermore, it is not clear from the comment what setting the number to 'null' entails.